### DISCUSSION TITLE
Can't delete a node label

### DISCUSSION THREAD

**Discussion by CAQ** [TuGraph-family/tugraph-db#82] [Q&A] [answered]
建模 => 顶点标签 => click one label => then on the right, click the 删除标签 button => prompts 删除失败
Log: 
: [StateMachine] [Calling Cypher] elapsed=7.93457031e-4, res=BAD_REQUEST, cypher=[CALL db.deleteLabel('node', '顶点标签名')]
: [RestServer] Illegal request: POST /cypher HTTP/1.1
...
: [RestServer] : Wrong argument given for label type: must be 'vertex' or 'edge'.

**Comment by knightast** [comment]
This is a known bug of TuGraph Browser, and we will fix in the next version

**Reply by CAQ** [reply]
Can I instead delete the label from command line?

**Reply by mumumi** [reply]
CALL db.deleteLabel('vertex', 'movie')

**Comment by KingSkyLi** [comment, marked as answer]
可视化建模部分删除node label报错的问题，已经在[https://github.com/TuGraph-db/tugraph-web/tree/v3.3.1](https://github.com/TuGraph-db/tugraph-web/tree/v3.3.1)仓库完成修复。
